.github/workflows/dispatch-build-bottle.yml

```diff
@@ -132,19 +132,12 @@ jobs:
         if: always() && runner.os == 'Linux'
         run: docker cp ${{github.sha}}:/tmp/bottles .
 
-      - name: Failures summary for brew test-bot --only-formulae
+      - name: Failures Summary and Bottle Result
```

Comment:
Not a fan of the way this combines the failures summary and `brew bottle` output into one step:

https://github.com/Homebrew/homebrew-core/runs/7869970862?check_suite_focus=true#step:7:1

I'm often interested in only one of those at any given point, and I'd rather only see what I'm looking for when I expand an item in the workflow output. (I imagine other maintainers have similar opinions, but feel free to point out if I'm mistaken here.)

Is there any way we can split these back up?

Reply:
@carlocab if this is worse for you: feel free to revert this.

Reply:
One option we could implement is to instead group `brew test-bot' and its following result. I thought `steps_output.txt` and `bottle_output.txt` were products of the same `brew test-bot` steps so it's okay to group them.

Reply:
`steps_output.txt` is just a duplicate of the tail of the step that preceded it (the step with `id: brew-test-bot-formulae`). To see this, compare the [tail of `brew-test-bot-formulae`](https://github.com/Homebrew/homebrew-core/runs/7869970862?check_suite_focus=true#step:6:5312) with the [contents of `steps_output.txt`](https://github.com/Homebrew/homebrew-core/runs/7869970862?check_suite_focus=true#step:7:27).

Dumping it anyway is useful because it gives you a way of viewing that tail without a lot of extraneous information. Putting it with other things defeats much of the purpose of creating `steps_output.txt`.

> One option we could implement is to instead group `brew test-bot` and its following result.

Could you elaborate on what you had in mind?

Reply:
I've opened https://github.com/Homebrew/actions/pull/295.